File: responses/VBF38S-2X2/2_reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    # write your code in Python
    a = int(x % 3)
   # print(a)
    if a == 0:
        e = x - 3
    else:    
        e = x - a
    s = 3
    
    r = int((e * (s + e))/6)
   # print(r)
    
    a = int(x % 5)
    if a == 0:
        e = x - 5
    else:    
        e = x - a
    s = 5
    
    r = r + int((e * (s + e))/10)
    return r
    
    
    

File: responses/9659ZU-ATX/2_reverse_digits.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(x)
  # write your code in Ruby
  converter_integer = x.to_s
  converter_integer[0].eql?('-') ? "-#{x.to_s.reverse}".to_i : x.to_s.reverse.to_i
  
end